src/components/main_todos-list/main_todos-list.tsx

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+import cn from 'classnames';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  visibleTodos: Todo[],
+};
+
+export const Main: React.FC<Props> = ({
+  visibleTodos,
+}) => {
+  return (
+    <section className="todoapp__main">
+      {/* This is a completed todo */}
+      {visibleTodos.map(todo => {
+        return (
+          <div
+            className={cn(
+              'todo',
+              { completed: todo.completed },
+            )}
+            key={todo.id}
+          >
+            <label className="todo__status-label">
+              <input
+                type="checkbox"
+                className="todo__status"
+              />
+            </label>
+
+            <span className="todo__title">{todo.title}</span>
+            <button type="button" className="todo__remove">×</button>
+
+            <div className="modal overlay">
+              <div className="modal-background has-background-white-ter" />
+              <div className="loader" />
+            </div>
+          </div>
+        );
+      })}
+      <div className="todo completed">
+        <label className="todo__status-label">
+          <input
+            type="checkbox"
+            className="todo__status"
+            checked
+          />
+        </label>
```

Comment:
```suggestion
     
```
here and below remove redundant jsx
it is just examples for todo.
should be only list of todos from server